samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1562,6 +1562,34 @@
           }
         ]
       }
+    },
+    "StepFunctionsExecutionPolicy": {
+      "Description": "Gives permission to start a Step Functions state machine execution",
+      "Parameters": {
+        "StateMachineName": {
+          "Description":"The name of the state machine to execute."
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "states:StartExecution"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:${parameterName}",
```

Comment:
Change `parameterName` to `stateMachineName` for better readability. Don't forget to also change it on line 1584.